frontend/src/components/PopupComponent.vue

```diff
@@ -0,0 +1,11 @@
+<template>
```

Comment:
I think it would be better to have an x in the top right corner to close the popup and not have each component have its own button for that